### DISCUSSION TITLE
Viessmann Vitodens 200-W B2HF-19

### DISCUSSION THREAD

**Discussion by HelmutFi** [openv/openv#528] [General]
Ich hab voriges Jahr eine neue Vitodens 200-W (Gasbrennwert, Wandmontage) bekommen. Diese hat das Modul mit dem integrierten WLAN. 
Derzeit kann ich die Anlage über IPS (IP Symcon) nur über die Cloud API von Viessmann steuern. Das funktioniert auch meistens. Aber zum einen fällt das immer wieder aus und wichtige Werte fehlen einfach.
Würde das OptoLink Modul mit dieser Anlage auch funktionieren?
Vielen Dank
Helmu

**Comment by speters** [comment]
Markus Fritze hat in [VitosoftCommunication](https://github.com/sarnau/InsideViessmannVitosoft/blob/main/VitosoftCommunication.md) ein paar Details zum WLAN Protokoll beschrieben.

Das würde ich an deiner Stelle zuerst ausprobieren.
Ggf. reicht [socat](http://www.dest-unreach.org/socat/), um die Kommunikation an einen "virtuellen" seriellen Port zu leiten, um dann mit den gängigen Softwares weiter zu arbeiten.
Wenn die o.g. Heizung eine Optolink-Schnittstelle besitzt, dann kann man das natürlich auch darüber machen, da führen mehrere Wege zum Ziel.

[Issue510](https://github.com/openv/openv/issues/510#issuecomment-1002964111)

**Comment by HelmutFi** [comment]
Danke. Ob die Anlage eine Optolink-Schnittstelle hat, oder nicht kann ich so jetzt nicht sagen. Ich denke aber nicht. Ich hab heute mal das Modul (das mit dem Display) abgenommen und mir die Stelle genauer angesehen, wo es sein könnte. Habe aber nichts gefunden. Dahinter ist nichts. Nur an der Stelle vom Display und den Softtouchbedientasten. 
Was ich allerdings noch gefunden hab, ist ein zweiter zweireihiger Stecker. Ähnlich bzw. ident dem, mit dem die Steuerung mit der Anlage verbunden ist.
